wallet/udb/treasury.go

```diff
@@ -31,6 +43,25 @@ func SetTSpendPolicy(dbtx walletdb.ReadWriteTx, hash *chainhash.Hash,
 	return b.Put(hash[:], []byte{byte(policy)})
 }
 
+// SetVSPTSpendPolicy sets a tspend vote policy for a specific tspend
+// transaction hash for a VSP customer's ticket hash.
+func SetVSPTSpendPolicy(dbtx walletdb.ReadWriteTx, ticketHash, tspendHash *chainhash.Hash,
+	policy stake.TreasuryVoteT) error {
+
+	k := make([]byte, 64)
+	copy(k, ticketHash[:])
+	copy(k[32:], tspendHash[:])
+
+	b := dbtx.ReadWriteBucket(vspTspendPolicyBucketKey)
+	if policy == stake.TreasuryVoteInvalid {
+		err := b.Delete(k)
+		if err != nil {
+			return errors.E(errors.IO, err)
+		}
```

Comment:
Was there supposed to be a `return` here (so that passing `TreasuryVoteInvalid` removes the per-ticket entry)? Otherwise this delete seems redundant because the `Put` below is going to get executed anyway.

Reply:
yes